airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -385,175 +383,296 @@ static Optional<ContainerOrchestratorConfig> getContainerOrchestratorConfig(fina
     }
   }
 
-  private static void launchWorkerApp(final Configs configs, final DSLContext configsDslContext, final DSLContext jobsDslContext) throws IOException {
-    MetricClientFactory.initialize(MetricEmittingApps.WORKER);
+  private static AirbyteApiClient getApiClient(final Configs configs) {
+    final var authHeader = configs.getAirbyteApiAuthHeaderName();
+
+    // control plane workers communicate with the Airbyte API within their internal network, so https
+    // isn't needed
+    final var scheme = configs.isControlPlaneWorker() ? "http" : "https";
```

Comment:
![image](https://user-images.githubusercontent.com/7798112/186285869-fe849f03-fa26-47a3-b7af-d3b64d381894.png)

@evantahler yeah the way we configure control plane workers is to use internal K8s DNS to communicate with the API service, all http. Let me know if that doesn't actually answer your question, hopefully I'm interpreting correctly